pkg/client/clientset/versioned/typed/nsx.vmware.com/v1alpha1/nsxserviceaccount.go

```diff
@@ -1,182 +0,0 @@
-/* Copyright © 2024 VMware, Inc. All Rights Reserved.
```

Comment:
Is this file no longer needed? Who is the consumer of this clientset/informer/lister?